### PR TITLE
COTECH-738 updated MobileArticleVideoPlayer.tsx

### PR DESCRIPTION
**Pull request by David-Iacovone**

### CODE REVIEW

File: src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx [hunk 1]
```diff
@@ -75,21 +83,18 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
...
 
 	return (
 		<>
 			<div ref={ref} className={clsx(styles.mobileArticleVideoTopPlaceholder, isScrollPlayer && `is-on-scroll-active`)}>
 				<MobileArticleVideoWrapper isScrollPlayer={isScrollPlayer}>
-					{adComplete && (
+					{jwpAdsSetupComplete.strategyRulesEnabled && (
 						<>
-							<JwPlayerWrapper
+							<StrategyRulesWrapper
```
Comment (latata) [inline src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx:94]: I think we should still render JwPlayerWrapper if `strategyRulesEnabled` is `false`, right?

Comment (David-Iacovone) [reply src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx:94]: right, I initially thought we should substitute one with the other. But that's indeed what nandy's PR suggesting. Let me fix it.

File: src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx [hunk 2]
```diff
@@ -75,27 +85,40 @@ export const MobileArticleVideoPlayerContent: React.FC<MobileArticleVideoPlayerP
...
 								getDismissed={getDismissed}
 								config={getArticleVideoConfig(videoDetails)}
-								onReady={(playerInstance) => {
-									articlePlayerOnReady(videoDetails, playerInstance);
-									setIsPlayerReady(true);
-								}}
+								onReady={onPlayerInstanceReady}
 								stopAutoAdvanceOnExitViewport={false}
```
Comment (latata) [inline src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx:100]: I think it's not needed for StrategyRulesWrapper

**Review by latata — APPROVED** [review]

File: src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx (hunk 2 above)
Comment (David-Iacovone) [reply src/jwplayer/players/MobileArticleVideoPlayer/MobileArticleVideoPlayer.tsx:100]: It was implemented this way on desktop so wanted to be consistent

**Pull request merged**